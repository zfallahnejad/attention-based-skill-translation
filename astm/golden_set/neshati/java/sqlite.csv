11654,sqlite
101361,sqlite
115145,sqlite
179850,sqlite
1267661,sqlite
2649012,sqlite
