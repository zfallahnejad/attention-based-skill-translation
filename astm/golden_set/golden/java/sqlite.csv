11654,86,79,0.5212121212121212
101361,68,69,0.49635036496350365
115145,40,10,0.8
179850,23,9,0.71875
1267661,21,12,0.6363636363636364
2649012,16,12,0.5714285714285714
844882,10,6,0.625
22656,10,1,0.9090909090909091
